migration_toolkit_for_containers/advanced-migration-options-mtc.adoc

```diff
@@ -43,6 +43,7 @@ include::modules/migration-mapping-destination-namespaces-in-the-migplan-cr.adoc
 include::modules/migration-excluding-pvcs.adoc[leveloffset=+2]
 include::modules/migration-mapping-pvcs.adoc[leveloffset=+2]
 include::modules/migration-editing-pvs-in-migplan.adoc[leveloffset=+2]
+include::modules/migration-converting-the-storage-class-of-persistent-volumes-cam.adoc[leveloffset=+2]
```

Comment:
Location looks OK. File name should not contain 'cam'. I also think "migration-converting-storage-classes.adoc" would be a better file name.